script/check_newsfragments.py

```diff
@@ -39,7 +39,7 @@
 ISSUES_KEYWORDS = "|".join(VALID_ISSUES_KEYWORDS)
 VALID_CHANGELOG_PATTERN = (
     rf"(?P<description>(.*\n)*(.*\.\n))\n(?P<ref>({ISSUES_KEYWORDS})"
-    r" (PyCQA/astroid)?#(?P<issue>\d+))"
+    r" (pylint-dev/astroid)?#(?P<issue>\d+))"
```

Comment:
Huho, this is a regex
```suggestion
    r" (pylint\-dev/astroid)?#(?P<issue>\d+))"
```